### PR TITLE
fix: upgrade failed in multi-system mode

### PR DESCRIPTION
**Pull request by l631197874**
/boot partition free space overflow

Log: not backup /boot/efi
Issue: https://github.com/linuxdeepin/developer-center/issues/4168

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+deepin-system-upgrade (6.0.6) unstable; urgency=medium
+
+  * modify upgrade failed in multi-system.
+  *
```
Comment (zsien) [inline debian/changelog:4]: 多了一行

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+deepin-system-upgrade (6.0.6) unstable; urgency=medium
+
+  * modify upgrade failed in multi-system.
+  * 
+
+ -- Xin Lin <far@far-PC>  Thu, 18 May 2023 16:10:24 +0800
```
Comment (zsien) [inline debian/changelog:6]: 邮箱有问题

**Review by zsien — CHANGES_REQUESTED** [review]

File: debian/changelog (hunk 1 above)
Comment (l631197874) [reply debian/changelog]: done

File: debian/changelog (hunk 2 above)
Comment (l631197874) [reply debian/changelog]: done

**Review by Decodetalkers — APPROVED** [review]

**Review by zsien — APPROVED** [review]

**Pull request merged**